packages/mdc-menu/simple/foundation.js

```diff
@@ -300,6 +301,14 @@ export default class MDCSimpleMenuFoundation extends MDCFoundation {
   }
 
   handlePossibleSelected_(evt) {
+    const targetIsDisabled =
+      this.adapter_.getAttributeForEventTarget(
+        evt.target, MDCSimpleMenuFoundation.strings.ARIA_DISABLED_ATTR
+      ) === 'true';
+    if (targetIsDisabled) {
+      evt.stopPropagation();
```

Comment:
The other option is to pass the event to the close method and check if the target is disabled there. The problem is that there is a document click handler that closes the select that gets triggered if we let the event propagate.